dcos_test_utils/onprem.py

```diff
@@ -60,10 +60,9 @@ def get_public_agent_ips(self):
         return copy.copy(self.public_agents)
 
     @classmethod
-    def from_hosts(cls, ssh_client, hosts, num_masters, num_private_agents, num_public_agents):
-        bootstrap_host, masters, private_agents, public_agents = (
-            cls.partition_cluster(hosts, num_masters, num_private_agents, num_public_agents))
-
+    def from_hosts(cls, ssh_client, bootstrap_host, cluster_hosts, num_masters, num_private_agents, num_public_agents):
```

Comment:
being as we do not have the ability to split out a bootstrap host in all providers (gce is where this breaks down), I think we should:
* leave hosts as hosts ( do not change to `cluster_hosts`)
* make bootstrap_host an optional argument argument. if it is None, sort the list and pop off the first item